interface/main/calendar/add_edit_event.php

```diff
@@ -1904,7 +1912,7 @@ function validateform(event,valu){
     }
     ?>
 
-    var submit = submitme(1, event, 'theform', collectvalidation);
+    var submit = submitme(1, event, '<?php echo $form_id; ?>', collectvalidation);
```

Comment:
Drop the quotes and use <?php echo js_escape($form_id); ?> here (it will auto add the quotes).  Since this file is so huge even though you've defined constants 500 lines above its safer to escape it here in case anyone modifies it between the locality you defined this variable, and the locality it is used.